x/ethbridge/types/msgs.go

```diff
@@ -671,6 +672,20 @@ func (msg MsgSetBlacklist) GetSignBytes() []byte {
 }
 
 func (msg *MsgSetBlacklist) ValidateBasic() error {
+	if msg.From == "" {
+		return sdkerrors.Wrap(sdkerrors.ErrInvalidAddress, msg.From)
+	}
+
+	if len(msg.Addresses) == 0 {
+		return ErrEmptyBlackList
```

Comment:
Why isnt an empty blacklist allowed? For example, when we init, or when we want to clear the list

Reply:
It is my wrong assumption, I thought it is an incremental function. will remove this check.

Reply:
Can you add a test such to verify empty list wont throw err?

Reply:
sure, will add it.